penny_university/management/commands/background_tasks.py

```diff
@@ -0,0 +1,19 @@
+from background_task.management.commands.process_tasks import Command as ProcessTasks
+from background_task.utils import SignalManager
+from django.conf import settings
+
+from bot.tasks import send_penny_chat_reminders
+
+
+class Command(ProcessTasks):
+    def handle(self, *args, **options):
+        send_penny_chat_reminders()
```

Comment:
Oh, I missed it right here. I'm still a bit confused about what needs the background decorator and what doesn't.

Reply:
The previous `process_tasks` command (which is currently running in production) turns on the task queue that I installed. This command will replace it. It will effectively serve the same purpose, but before it start it will also quickly check the database for penny_chats that need reminders and fire them off. After that it will be in `process_tasks` mode. I'm using [heroku's scheduler](https://devcenter.heroku.com/articles/scheduler) to run this command every 10 minutes.

Eventually `send_penny_chat_reminders` will become time consuming enough that we'll have to split them out again... but that will be quite some time.